README.md

```diff
@@ -117,28 +117,16 @@ archive { '/tmp/test100k.db':
 
 ### Puppet URL
 
-Below is an example of how to deploy a tar.gz to a local directory when it is
-served via the ```puppet:///``` style url which is not currently supported.
+Since march 2017, the Archive type also supports puppet url's. Here is an example
```

Comment:
So not sure what happens if the puppet:/// url points to a directory v.s. a file. Maybe we should clarify it in documentation about expectations if it doesn't handle directory.